Server/mods/deathmatch/logic/CDummy.cpp

```diff
@@ -48,12 +48,40 @@ void CDummy::Unlink()
     }
 }
 
+void CDummy::GetRotationDegrees(CVector& vecRotation)
+{
+    GetRotationRadians(vecRotation);
+    ConvertRadiansToDegrees(vecRotation);
+}
+
+void CDummy::SetRotationDegrees(const CVector& vecRotation)
+{
+    CVector vecTemp = vecRotation;
+    ConvertDegreesToRadians(vecTemp);
+    SetRotationRadians(vecTemp);
+}
+
+void CDummy::GetMatrix(CMatrix& matrix)
+{
+    matrix = CMatrix(m_vecPosition, m_vecRotation);
+}
+
+void CDummy::SetMatrix(const CMatrix& matrix)
+{
+    m_vecRotation = matrix.GetRotation();
+    m_vecPosition = matrix.GetPosition();
+}
+
 bool CDummy::ReadSpecialData(const int iLine)
 {
     // Grab the position data
     GetCustomDataFloat("posX", m_vecPosition.fX, true);
     GetCustomDataFloat("posY", m_vecPosition.fY, true);
     GetCustomDataFloat("posZ", m_vecPosition.fZ, true);
 
+    // Grab the rotation data
+    GetCustomDataFloat("rotX", m_vecRotation.fX, true);
+    GetCustomDataFloat("rotY", m_vecRotation.fY, true);
+    GetCustomDataFloat("rotZ", m_vecRotation.fZ, true);
```

Comment:
Might break something?
Perhaps add a version check maybe. Not sure.

Reply:
I will check what it does I wasn't sure at all so I just copied to make it look consistent

Reply:
It's map loading stuff AFAIK. Although, I think map elements probably have their position and rotation set to 0 by default, so it shouldn't be an issue.

Reply:
So, I should remove these 3 lines, or keep them?